testsuite/features/reposync/srv_sync_channels.feature

```diff
@@ -11,38 +11,38 @@ Feature: List available channels and enable them
-    And I should get "    [ ] SLE15-SP4-Installer-Updates for x86_64 SUSE Linux Enterprise Server 15 SP4 x86_64 [sle15-sp4-installer-updates-x86_64]"
+    Then I should get "[ ] SLE-Product-SLES15-SP7-Pool for x86_64 SUSE Linux Enterprise Server 15 SP7 x86_64 [sle-product-sles15-sp7-pool-x86_64]"
+    And I should get "    [ ] SLE-Product-SLES15-SP7-Updates for x86_64 SUSE Linux Enterprise Server 15 SP7 x86_64 [sle-product-sles15-sp7-updates-x86_64]"
+    And I should get "    [ ] SLE15-SP7-Installer-Updates for x86_64 SUSE Linux Enterprise Server 15 SP7 x86_64 [sle15-sp7-installer-updates-x86_64]"
 
   Scenario: List available mandatory channels
     When I execute mgr-sync "list channels -e --no-optional"
-    Then I should get "[ ] SLE-Product-SLES15-SP4-Pool for x86_64 SUSE Linux Enterprise Server 15 SP4 x86_64 [sle-product-sles15-sp4-pool-x86_64]"
-    And I should get "    [ ] SLE-Product-SLES15-SP4-Updates for x86_64 SUSE Linux Enterprise Server 15 SP4 x86_64 [sle-product-sles15-sp4-updates-x86_64]"
-    And I shouldn't get "    [ ] SLE15-SP4-Installer-Updates for x86_64 SUSE Linux Enterprise Server 15 SP4 x86_64 [sle15-sp4-installer-updates-x86_64]"
+    Then I should get "[ ] SLE-Product-SLES15-SP7-Pool for x86_64 SUSE Linux Enterprise Server 15 SP7 x86_64 [sle-product-sles15-sp7-pool-x86_64]"
+    And I should get "    [ ] SLE-Product-SLES15-SP7-Updates for x86_64 SUSE Linux Enterprise Server 15 SP7 x86_64 [sle-product-sles15-sp7-updates-x86_64]"
+    And I shouldn't get "    [ ] SLE15-SP7-Installer-Updates for x86_64 SUSE Linux Enterprise Server 15 SP7 x86_64 [sle15-sp7-installer-updates-x86_64]"
 
   Scenario: List Server product
     When I execute mgr-sync "list products"
-    Then I should get "[ ] SUSE Linux Enterprise Server 15 SP4 x86_64"
+    Then I should get "[ ] SUSE Linux Enterprise Server 15 SP7 x86_64"
 
 @proxy
 @susemanager
   Scenario: List Proxy product
-    When I execute mgr-sync "list products"
-    Then I should get "[ ] SUSE Manager Proxy 4.3 x86_64"
+    When I execute mgr-sync "list products --expand"
+    Then I should get "[ ] SUSE Multi-Linux Manager Proxy Extension for SLE 5.1 x86_64"
 
 @proxy
 @susemanager
   Scenario: List all products for SUSE Multi-Linux Manager
     When I execute mgr-sync "list products --expand"
-    Then I should get "[ ] SUSE Linux Enterprise Server 15 SP4 x86_64"
-    And I should get "[ ] SUSE Manager Proxy 4.3 x86_64"
-    And I should get "  [ ] (R) SUSE Multi-Linux Manager Beta Client Tools for SUSE Liberty Linux 7, RHEL and clones 7 x86_64 (BETA)"
-    And I should get "  [ ] (R) SUSE Multi-Linux Manager Beta Client Tools for SLE 15 x86_64 (BETA)"
+    Then I should get "[ ] SUSE Linux Enterprise Server 15 SP7 x86_64"
+    And I should get "  [ ] SUSE Multi-Linux Manager Proxy Extension 5.1 x86_64"
+    And I should get "  [ ] (R) SUSE Multi-Linux Manager Client Tools for SUSE Liberty Linux 7, RHEL and clones 7 x86_64"
```

Comment:
Oh wow, I never noticed it `(R) `
That's expected? I think it's worth to ask about it.